internal/repository/integration/employee_test.go

```diff
@@ -82,20 +86,44 @@ func DeleteData(posRepo domain.PositionsRepository, empRepo domain.EmployeesRepo
 	return errs
 }
 
-func TestEmployeeRepository_Create(t *testing.T) {
+func initEmpRepo() (domain.EmployeesRepository, domain.PositionsRepository, error) {
 	config, err := conf.NewConfig()
 	if err != nil {
 		log.Fatalf("Error while getting config: %s", err)
 	}
 
-	db, err := database.ConnectPostgres(config.DB)
-	if err != nil {
-		log.Fatalf("Connection to database failed: %s", err)
+	var employeeRepo domain.EmployeesRepository
+	var positionRepo domain.PositionsRepository
+
+	switch config.DatabaseType {
+	case "postgres":
```

Comment:
I would suggest having integration tests separated from one another, so they won't overlap.
So structure would be smth like 
```text
integration
integration/mongo/employee_test.go (or integration/mongo_employee_test.go)
integration/postgres/employee_test.go (or integration/postgres_employee_test.go)
...
```

Reply:
will it be ok, if only one type of repo can be tested at once, since i am initializing config with only MongoConfig or PostgresConfig